app/scripts/directives/logViewer.js

```diff
@@ -27,6 +28,7 @@ angular.module('openshiftConsole')
              DataService,
              HTMLService,
              ModalsService,
+             AggregatedLoggingService,
```

Comment:
This is not defined in this repo.  Please move to this repo per @spadgett request

Reply:
It's in vendor.js

Reply:
You are missing the [point](https://github.com/openshift/origin-web-common/pull/342#issuecomment-415014888).  The intention is not do any other release of origin-web-common.  All the files in 'dist' are generated files that are vendored in through an automated process.  The recommendation is o close #342 and move the [file](https://github.com/openshift/origin-web-common/pull/342/files#diff-de4757cd302ff949e0ec3e237a13d05d) to origin-web-console